### PR TITLE
passing experimental

### PR DESCRIPTION
**Pull request by 64-B1T**
This is the preliminary pull for the mini-collision checking libary designed to integrate with the rest of my code.

### CODE REVIEW

File: collision_manager.py [hunk 1]
```diff
@@ -62,6 +64,7 @@ def createMesh(position, file_name, type='stl'):
         trimesh.Trimesh: new trimesh mesh
 
     """
+    print(file_name)
```
Comment (mcevoyandy) [inline collision_manager.py]: needed? add context to logging msg or remove

File: collision_manager.py [hunk 2]
```diff
@@ -256,22 +273,39 @@ def checkInternalCollisions(self):
...
+            if self.ignore_ee:
+                if first == self.arm.link_names[-1] or second == self.arm.link_names[-1]:
+                    continue
             link_index = self.arm.link_names.index(first)
             if link_index > 0 and self.arm.link_names[link_index - 1] == second:
                 continue
             elif link_index < self.num_links - 1 and self.arm.link_names[link_index + 1] == second:
                 continue
             else:
+                #print(names)
```
Comment (mcevoyandy) [inline collision_manager.py]: remove

File: collision_manager.py [hunk 3]
```diff
@@ -256,22 +273,39 @@ def checkInternalCollisions(self):
...
 
+    def drawArmMeshes(self, ax):
+        """
+        Draw Arm Meshes (and update backup, displayable meshes)
+        Args:
+            ax: matplotlib axis object to draw to
+        """
+        joint_transforms = self.arm.getJointTransforms()
+        for i in range(self.num_links):
+            self.meshes[self.arm.link_names[i]].apply_transform(self.old_transforms[i].inv().gTM())
```
Comment (mcevoyandy) [inline collision_manager.py:308]: not clear what's going on here, why apply old_transform then joint_transform?

File: collision_manager.py (hunk 1 above)
Comment (64-B1T) [reply collision_manager.py]: Removed.

File: collision_manager.py (hunk 2 above)
Comment (64-B1T) [reply collision_manager.py]: Removed

File: collision_manager.py (hunk 3 above)
Comment (64-B1T) [reply collision_manager.py:308]: Descriptive comment added.

**Review by mcevoyandy — APPROVED** [review]

**Pull request merged**